.github/workflows/build.yml

```diff
@@ -34,6 +35,7 @@ jobs:
       - run: make test
   integration:
     runs-on: ubuntu-20.04
+    timeout-minutes: 30
```

Comment:
I think I've seen integration tests take 35 minutes occasionally. Can we do something like 40?